### PR TITLE
Complete Menubar items

### PR DESCRIPTION
**Pull request by yozhgoor**
I started with the menu bar, just added some icon and sub-menu

### CODE REVIEW

**Comment by cecton** [comment]
This looks already pretty good :+1:

**Comment by yozhgoor** [comment]
the advanced parameters sub-menu is good to go now

File: src/Menubar.tsx [hunk 1]
```diff
@@ -53,20 +55,119 @@ function MenubarSettings({ openPanel }: IPanelProps) {
...
+      <Radio label="Custom" value="custom" />
+    </RadioGroup>
+  );
+}
+
+function MenubarLightning() {
+  const [radioCheck, setRadioCheck] = useState("nightOutside");
+  return (
+
+    <RadioGroup
```
Comment (cecton) [inline src/Menubar.tsx]:
```suggestion
  return (
    <RadioGroup
```

File: src/Menubar.tsx [hunk 2]
```diff
@@ -53,20 +55,118 @@ function MenubarSettings({ openPanel }: IPanelProps) {
...
+      <Radio label="Flipped" value="flipped" />
+    </RadioGroup>
+  );
+}
+
+function MenubarAngle() {
+  const [radioCheck, setRadioCheck] = useState("extended");
+  return (
+
+    <RadioGroup
```
Comment (cecton) [inline src/Menubar.tsx]:
```suggestion
  return (
    <RadioGroup
```

File: src/Menubar.tsx [hunk 3]
```diff
@@ -53,20 +55,117 @@ function MenubarSettings({ openPanel }: IPanelProps) {
...
+      />
     </Menu>
   );
 }
 
 function MenubarDisplay() {
+  const [radioCheck, setRadioCheck] = useState("3dFlat");
+  return (
+
+    <RadioGroup
```
Comment (cecton) [inline src/Menubar.tsx]:
```suggestion
  return (
    <RadioGroup
```

File: src/Menubar.tsx [hunk 4]
```diff
@@ -53,20 +55,116 @@ function MenubarSettings({ openPanel }: IPanelProps) {
...
+        openPanel({ component : MenuAdvanced, title: "Advanced parameters"})
+        }
+      />
     </Menu>
   );
 }
 
 function MenubarDisplay() {
+  const [radioCheck, setRadioCheck] = useState("3dFlat");
+  return (
```
Comment (cecton) [inline src/Menubar.tsx]:
```suggestion
  const [radioCheck, setRadioCheck] = useState("3dFlat");

  return (
```

File: src/Menubar.tsx [hunk 5]
```diff
@@ -53,20 +55,116 @@ function MenubarSettings({ openPanel }: IPanelProps) {
...
+      <Radio label="Anaglyph" value="anaglyph" />
+      <Radio label="Inverted" value="inverted" />
+      <Radio label="Flipped" value="flipped" />
+    </RadioGroup>
+  );
+}
+
+function MenubarAngle() {
+  const [radioCheck, setRadioCheck] = useState("extended");
+  return (
```
Comment (cecton) [inline src/Menubar.tsx]:
```suggestion
  const [radioCheck, setRadioCheck] = useState("extended");

  return (
```

File: src/Menubar.tsx [hunk 6]
```diff
@@ -53,20 +55,116 @@ function MenubarSettings({ openPanel }: IPanelProps) {
...
+      <Radio label="Extended" value="extended" />
+      <Radio label="Square" value="square" />
+      <Radio label="Custom" value="custom" />
+    </RadioGroup>
+  );
+}
+
+function MenubarLightning() {
+  const [radioCheck, setRadioCheck] = useState("nightOutside");
+  return (
```
Comment (cecton) [inline src/Menubar.tsx]:
```suggestion
  const [radioCheck, setRadioCheck] = useState("nightOutside");

  return (
```

File: src/Menubar.tsx [hunk 7]
```diff
@@ -53,20 +55,116 @@ function MenubarSettings({ openPanel }: IPanelProps) {
...
+  <Menu>
+    <MenuItem icon="wrench" text="WB" />
+    <MenuItem icon="flash" text="Exposure" />
+    <MenuItem icon="contrast" text="Contrast" />
+    <MenuItem icon="delta" text="Sharpness" />
+    <MenuItem icon="pivot-table" text="Stabilization" />
+    <MenuItem icon="cell-tower" text="Av gain" />
+  </Menu>
+  );
```
Comment (cecton) [inline src/Menubar.tsx]:
```suggestion
  return (
    <Menu>
      <MenuItem icon="wrench" text="WB" />
      <MenuItem icon="flash" text="Exposure" />
      <MenuItem icon="contrast" text="Contrast" />
      <MenuItem icon="delta" text="Sharpness" />
      <MenuItem icon="pivot-table" text="Stabilization" />
      <MenuItem icon="cell-tower" text="Av gain" />
    </Menu>
  );
```

**Comment by cecton** [comment]
Minor annoyance: use `npm run fmt` next time

**Pull request merged**